src/vm/jvm/runtime/org/raku/rakudo/Binder.java

```diff
@@ -470,7 +470,7 @@ else if (Ops.istype_nd(decontValue, gcx.PositionalBindFailover, tc) != 0) {
                     /* Try to figure out the most helpful name for the expected. */
                     SixModelObject expectedType = null;
                     SixModelObject postConstraints = param.get_attribute_boxed(tc, gcx.Parameter,
-                            "$!post_contraints", HINT_post_constraints);
+                            "$!post_constraints", HINT_post_constraints);
```

Comment:
Good catch! This could even fix some problems, because ```postConstraints``` will always be null (same goes for the same typo in line 691).